python/tvm/autotvm/task/dispatcher.py

```diff
@@ -238,12 +246,12 @@ class ApplyHistoryBest(DispatchContext):
 
     Parameters
     ----------
-    records : str, list of str, or iterator of (autotvm.measure.MeasureInput,\
-                                                autotvm.measure.MeasureResult)
-        Collection of tuning records.
-        If is str, then it should be the filename of a records log file.
-        Each row of this file is an encoded record pair. If it is a list, it can either be
-        a list of paths to log files that will be loaded jointly or an iterator or records.
+    records : Records or iterator of Records objects, where a Records
```

Comment:
i think it's reasonable to allow for the variety of ways to read in records, but just wondering if there's a use case for supporting `Iterator[Records]`?

Reply:
I don't think supporting `Iterator[Records]`makes any sense, and would love to get rid of this functionality. However, the function `ApplyHistoryBest` currently supports taking as input an iterator of `str` paths, so removing this would break previously existing functionality. Thoughts @areusch?